IntelliTect.TestTools.Console/ConsoleAssert.cs

```diff
@@ -42,9 +66,40 @@ public static string Expect(string expected, Action action, bool normalizeLineEn
         /// <param name="action">Method to be run</param>
         /// <param name="args">Args to pass to the function.</param>
         /// <param name="normalizeLineEndings">Whether differences in line ending styles should be ignored.</param>
-        public static string Expect(string expected, Action<string[]> action, bool normalizeLineEndings = true, params string[] args)
+        [Obsolete("Use Expect with " + nameof(NormalizeOptions))]
+        public static string Expect(string expected, 
+            Action<string[]> action, 
+            bool normalizeLineEndings = true, 
+            params string[] args)
         {
-            return Expect(expected, ()=>action(args), (left, right) => left == right, normalizeLineEndings);
+            return Expect(expected, 
+                ()=>action(args), 
+                (left, right) => left == right, 
+                normalizeLineEndings ? NormalizeOptions.NormalizeLineEndings : NormalizeOptions.None);
```

Comment:
Regardless of whether `normalizeLineEndings` is true or false, shouldn't `NormalizeOptions.StripAnsiEscapeCodes` be set (rather than `NormalizeOptions.None`)?
In fact, consider a `NormalizeOptions.Default` enum value where `NormalizeOptions.Default = NormalizeOptions.NormalizeLineEndings | NormalizeOptions.StripAnsiEscapeCodes`.